.github/workflows/unit-tests.yml

```diff
@@ -227,12 +227,12 @@ jobs:
             script -e -c './phpunit --group tty' /dev/null
 
       - name: Run tests with SIGCHLD enabled PHP
-        if: "matrix.php == '8.1' && ! matrix.mode"
+        if: "matrix.php == '8.2' && ! matrix.mode"
         run: |
           mkdir build
           cd build
-          wget -q https://github.com/symfony/binary-utils/releases/download/v0.1/php-8.1.2-pcntl-sigchild.tar.bz2
-          tar -xjf php-8.1.2-pcntl-sigchild.tar.bz2
+          wget -q https://github.com/symfony/binary-utils/releases/download/v0.1/php-8.2.6-pcntl-sigchild.tar.bz2
```

Comment:
This needs to be created first.

Btw, now that we have the symfony-tools organization for our tooling repositories, maybe this repository should be moved there, to make it more clear that it is also part of our internal tooling not covered by BC.

Reply:
I disabled the job.
@derrabus I'm going to need your help to get this one :pray: